src/lib/datepicker/datepicker.spec.ts

```diff
@@ -190,6 +190,26 @@ describe('MdDatepicker', () => {
         });
       });
 
+      it('setting twice to the same selected value should update input and close calendar', () => {
```

Comment:
'clicking the currently selected date should close the calendar without firing selectedChanged'